src/style/panel.less

```diff
@@ -227,6 +227,10 @@
     opacity: 0;
 }
 
+.artboard-presets, .recent-files {
```

Comment:
This change is a fix for the panel height in no-doc state. Before, the two panels didn't expand to the full height of the window.